### PR TITLE
Remove reference to master branch

### PR DESCRIPTION
**Pull request by tuanngocnguyen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <a href="https://travis-ci.org/catalyst/moodle-auth_saml2">
-<img src="https://travis-ci.org/catalyst/moodle-auth_saml2.svg?branch=master">
```
Comment (keevan) [inline README.md:2]: I think this should be a github badge similar to changes made here?
https://github.com/catalyst/moodle-auth_saml2/pull/636/files

**Review by keevan — COMMENTED** [review]
One small thing otherwise looks good

File: README.md (hunk 1 above)
Comment (tuanngocnguyen) [reply README.md:2]: We don't have the github workflow for 27_34 So I still use the travis-ci. one

Comment (keevan) [reply README.md:2]: Ah okay. All good then.

**Pull request merged**